run_test.go

```diff
@@ -286,7 +287,9 @@ func testSucceedRun(t *testing.T, format string, concurrency int, expectedOutput
 
 	suiteCtxReg := regexp.MustCompile(`suite_context.go:\d+`)
 	expectedOutput = suiteCtxReg.ReplaceAllString(expectedOutput, `suite_context.go:0`)
+	log.Println("expected clean: " + expectedOutput)
```

Comment:
this should be removed and if the information is needed, the assertion should be updated with `expected`